StudCity.Frontend/src/UI/Avatar/index.jsx

```diff
@@ -1,9 +1,10 @@
 import React from "react";
+import { DEFAULT_AVATAR_URL } from "../../utils/constants";
 
 function Avatar({ src, className = "", children }) {
 	return (
 		<div className={`rounded-full mx-auto overflow-hidden ${className}`}>
-			<img src={src} className="w-full h-full object-contain" alt="" />
+			<img src={src || DEFAULT_AVATAR_URL} className="w-full h-full object-contain" alt="" />
```

Comment:
I think it might be better to put DEFAULT_AVATAR_URL as default value for src directly where you destructurize props